src/ott/geometry/graph.py

```diff
@@ -40,6 +40,7 @@ class Graph(geometry.Geometry):
     directed: Whether the ``graph`` is directed. If not, it will be made
       undirected as :math:`G + G^T`. This parameter is ignored when  directly
       passing the Laplacian, which is assumed to be symmetric.
+    normalize: Whether to normalize the Laplacian.
```

Comment:
maybe a few more comments on what is meant by normalizing a Laplacian.

Reply:
Done.